plugins/obs-ffmpeg/data/locale/it-IT.ini

```diff
@@ -26,6 +26,7 @@ Looping="Ripeti"
 InputFormat="Formato dell'input"
 BufferingMB="Buffering della rete"
 HardwareDecode="Utilizza la decodifica hardware quando disponibile"
+ForceTCP="Forza l'utilizzo del TCP con RTSP"
```

Comment:
Translations will be overwritten/added in the next translations push, so you should revert this change here.